src/components/page/head.vue

```diff
@@ -39,6 +42,12 @@ defineOptions({
   margin-top: 0;
 }
 
+#page-head-title-infonav {
+  padding: 10px 0px;
+  display: flex;
+  align-items: center;
+}
```

Comment:
flex-shrink around the infonav slot contents? important if title is long and unbroken with no wrap opportunities.
`#page-head-title ~ * { flex-shrink: 0; }` or `#page-head .infonav { flex-shrink: 0; }`

Reply:
it doesnt seem necessary when making the other part of the page title wrap